README.md

```diff
@@ -36,7 +36,7 @@ Then open vim to a .cs file and install the stdio server with `:OmniSharpInstall
 * Find implementations/derived types
 * Find usages
 * Contextual code actions (unused usings, use var....etc.) (can use plugin: [fzf.vim](https://github.com/junegunn/fzf.vim), [CtrlP](https://github.com/ctrlpvim/ctrlp.vim) or [unite.vim](https://github.com/Shougo/unite.vim))
-* Find code issues (unused usings, use base type where possible....etc.) (requires plugin: [ALE](https://github.com/w0rp/ale) or [Syntastic](https://github.com/vim-syntastic/syntastic))
+* Find code issues in current file as well as globally (unused usings, use base type where possible....etc.) (requires plugin: [ALE](https://github.com/w0rp/ale) or [Syntastic](https://github.com/vim-syntastic/syntastic))
```

Comment:
This feature actually _doesn't_ require a linting plugin. Perhaps mention the global functionality as a separate bullet point, and mention that it populates the quickfix?

Reply:
Good point. I updated the readme